diff -Nur git-1.8.3/Makefile git-1.8.3.n/Makefile
--- git-1.8.3/Makefile	2013-05-24 21:15:35.000000000 +0200
+++ git-1.8.3.n/Makefile	2013-06-01 22:23:27.000000000 +0200
@@ -380,7 +380,7 @@
 gitwebdir = $(sharedir)/gitweb
 localedir = $(sharedir)/locale
 template_dir = share/git-core/templates
-htmldir = $(prefix)/share/doc/git-doc
+htmldir = share/doc/git-core
 ETC_GITCONFIG = $(sysconfdir)/gitconfig
 ETC_GITATTRIBUTES = $(sysconfdir)/gitattributes
 lib = lib
@@ -1061,7 +1061,7 @@
 	ifdef NO_R_TO_GCC_LINKER
 		# Some gcc does not accept and pass -R to the linker to specify
 		# the runtime dynamic library path.
-		CC_LD_DYNPATH = -Wl,-rpath,
+		CC_LD_DYNPATH = -L
 	else
 		CC_LD_DYNPATH = -R
 	endif
